src/Orleans.Core/Messaging/BufferPool.cs

```diff
@@ -1,80 +1,45 @@
 using System;
-using System.Collections.Concurrent;
+using System.Buffers;
 using System.Collections.Generic;
-using System.Threading;
+using System.Runtime.CompilerServices;
 using Orleans.Configuration;
 namespace Orleans.Runtime
 {
     internal class BufferPool
     {
+        private const int MaximumBufferSize = int.MaxValue;
```

Comment:
Is this unused?

Reply:
Yes, forgot to delete it. This is for `MemoryPool<T>.MaxBufferSize` which I'm planning to implement.